content/docs/guides/revocation.md

```diff
@@ -14,4 +14,67 @@ toc = false
 top = false
 +++
 
-## Revocation identifiers
\ No newline at end of file
+## Revocation identifiers
+
+Biscuit tokens are bearer tokens. Revoking bearer tokens is usually done through revocation lists: a list of
+tokens that are no longer accepted is shared with all verifying parties. When authorizing a biscuit token,
+the library will make sure the token has not been revoked.
+
+Such a mechanism relies on being able to uniquely identify tokens: we want to be able to revoke only the tokens
+that are not valid anymore, without revoking other tokens (even tokens with the same payload but have been issued
+to another holder). With offline attenuation, biscuits introduce another constraint: revoking a token should also
+revoke all derived tokens (else it would be trivial to circument revocation).
+
+The biscuit spec (and libraries) does not mandate _how_ to handle revocation lists. This is highly dependent on how
```

Comment:
maybe clarify that the *how* is about the infrastructure, deciding whether to load revocation ids in the authorizer to check them there or extract revocation ids from the authorizer and checking tem elsewhere, and how revocation lists are propagated in the system. The idea would be to give enough context so users can implement that part correctly (storing revocation ids with ttl per user, etc)

Reply:
updated for clarity. I did not mention specifics of advanced use cases